source/edge-server/connect.txt

```diff
@@ -63,12 +63,49 @@ documentation for your preferred SDK:
 - React Native SDK: :ref:`react-native-connect-to-mongodb-realm-backend-app`
 - Swift SDK: :ref:`ios-init-appclient`
 
+Supported Authentication Providers
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Edge Server currently supports a subset of authentication providers when
+connecting with an SDK:
+
+- :ref:`anonymous-authentication`
+- :ref:`email-password-authentication`
+- :ref:`api-key-authentication`
+- :ref:`custom-function-authentication`
```

Comment:
We do not support custom function authentication but we do support custom JWT support.

Reply:
Actually, I need to confirm this, custom function authentication might actually work. But to confirm, custom JWT does work.

Reply:
I don't think I've seen anything indicating that custom function auth is supposed to be supported - I think I just grabbed the wrong reference here. Updating it to custom JWT!